submission/persistence-mongo/src/test/kotlin/ac/uk/ebi/biostd/persistence/doc/db/converters/to/SectionConverterTest.kt

```diff
@@ -0,0 +1,171 @@
+package ac.uk.ebi.biostd.persistence.doc.db.converters.to
+
+import ac.uk.ebi.biostd.persistence.doc.model.DocAttribute
+import ac.uk.ebi.biostd.persistence.doc.model.DocFile
+import ac.uk.ebi.biostd.persistence.doc.model.DocFileList
+import ac.uk.ebi.biostd.persistence.doc.model.DocFileTable
+import ac.uk.ebi.biostd.persistence.doc.model.DocLink
+import ac.uk.ebi.biostd.persistence.doc.model.DocLinkTable
+import ac.uk.ebi.biostd.persistence.doc.model.DocSection
+import ac.uk.ebi.biostd.persistence.doc.model.DocSectionTable
+import arrow.core.Either
+import ebi.ac.uk.util.collections.second
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.junit5.MockKExtension
+import org.assertj.core.api.Assertions.assertThat
+import org.bson.Document
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+
+@ExtendWith(MockKExtension::class)
+internal class SectionConverterTest(
+    @MockK val attributeConverter: AttributeConverter,
```

Comment:
Same comment about defining the mocks where they're actually used unless they're global